agent/provider/src/cli/rule.rs

```diff
@@ -1,30 +1,64 @@
-use crate::rules::{CertRule, OutboundRule};
-use crate::{
-    rules::{Mode, RulesManager},
-    startup_config::ProviderConfig,
-};
-use anyhow::Result;
+use crate::rules::outbound::{CertRule, Mode, OutboundRules};
+use crate::rules::restrict::{RestrictRule, RuleAccessor};
+use crate::rules::OutboundRule;
+use crate::{rules::RulesManager, startup_config::ProviderConfig};
+
+use anyhow::{bail, Result};
+use serde_json::Value;
 use std::collections::HashMap;
 use std::path::PathBuf;
 use structopt::StructOpt;
 use strum::VariantNames;
+
+use ya_client_model::NodeId;
 use ya_manifest_utils::keystore::{AddParams, AddResponse, Keystore};
 use ya_manifest_utils::short_cert_ids::shorten_cert_ids;
+use ya_manifest_utils::CompositeKeystore;
 use ya_utils_cli::{CommandOutput, ResponseTable};
 
 #[derive(StructOpt, Clone, Debug)]
 pub enum RuleCommand {
     /// Set Modes for specific Rules
     Set(SetRule),
+    /// Add new rule.
+    Add(AddRule),
+    /// Remove existing rule.
+    Remove(RemoveRule),
+    /// Enable all rules in category.
+    Enable(RuleCategory),
+    /// Disable all rules in category.
+    Disable(RuleCategory),
     /// List active Rules and their information
     List,
 }
 
+/// Left for compatibility only. Should be replaced by AddRule and RemoveRule.
 #[derive(StructOpt, Clone, Debug)]
 pub enum SetRule {
     Outbound(SetOutboundRule),
 }
 
+#[derive(StructOpt, Clone, Debug)]
+pub enum RuleCategory {
+    Outbound,
+    Blacklist,
```

Comment:
Naming it `Blocklist` would not help avoiding confusion with `whitelist` command?